libs/localization/src/lib/en.json

```diff
@@ -345,6 +345,7 @@
     "instanceView": "Instance View",
     "localExplanationView": "Local explanation",
     "noFeature": "No feature",
+    "scaleWarning": "For scale reasons of the UI, explanations are randomly downsampled",
```

Comment:
nit: is the comma meant to be there? I'm actually not entirely sure...

Reply:
oh maybe I can remove it